bin/local-env/docker-compose.yml

```diff
@@ -1,5 +1,3 @@
-version: '3.1'
```

Comment:
This is no longer necessary and raises warnings in GHA
<img width="1124" alt="image" src="https://github.com/user-attachments/assets/805734b0-ea20-4efe-a22a-83f55a613eb0">